assets/charts/control-plane/calico/values.yaml

```diff
@@ -13,3 +13,5 @@ calico:
   podCIDR: 10.2.0.0/16
   networkEncapsulation: "ipipMode: Always"
   blockedMetadataCIDRs: []
+  # Lokomotive specific change
+  encryptPodsTraffic:
```

Comment:
```suggestion
  # Lokomotive specific change.
  encryptPodsTraffic: false
```

Reply:
should this be false, do we want encrption by default or not?

Reply:
I don't think we want to encrypt by default. @iaguis ?

Reply:
As this is experimental feature, I think this should be false by default.